demo/top_down_pose_tracking_demo_with_mmdet.py

```diff
@@ -73,6 +73,16 @@ def main():
         type=int,
         default=0,
         help='Category id for bounding box detection model')
+    parser.add_argument(
+        '--oks',
+        action='store_true',
+        default=False,
+        help='Using OKS similarity for tracking')
+    parser.add_argument(
+        '--euro',
+        action='store_true',
+        default=False,
+        help='Using One_Euro_Filter for smooding')
```

Comment:
You mean, smoothing?

Reply:
I'm sorry 😢 
I worked on it in a hurry and wrote it down according to the pronunciation.
Thank you for pointing it out.